test_files/str/trim.ab

```diff
@@ -1,7 +1,7 @@
 pub fun trim(text) {
-    return "$\{{nameof text}##*( )}"
+    return unsafe $echo "{text}" | xargs$
```

Comment:
This is not a valid `trim` function as it basically removes all spaces inside as well.

<img width="570" alt="Screenshot 2024-05-30 at 21 53 21" src="https://github.com/Ph0enixKM/Amber/assets/29208124/45167b64-c16d-411d-b482-21492332d0a4">

What's the motivation behind this change in the code?

Reply:
I have unified the `test_amber` macro.
As a result, the test using the `trim` function in the original `test_files/str/trim.ab` failed.
Therefore, I copied and pasted the definition of the `trim` function from `src/std/main.ab`.

Reply:
Oh okay then we have to fix the `trim` function in a separate issue